spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -116,18 +116,18 @@ def assert_configuration_script_with_survey_spec
     expect(system).to have_attributes(
       :name        => "Ansible-JobTemplate-Survey",
       :description => "Ansible-JobTemplate-Description",
-      :manager_ref => "155",
+      :manager_ref => "81",
       :variables   => {'abc' => 123}
     )
     survey = system.survey_spec
     expect(survey).to be_a Hash
-    expect(survey['spec'].first['index']).to eq 0
```

Comment:
Just checked. We can POST and `index` and it gets stored. But it does not change the order.

```json
{
    "description": "",
    "spec": [
        {
            "index": 1,
            "required": true,
            "min": null,
            "default": "",
            "max": null,
            "question_description": "Survey",
            "choices": "",
            "variable": "test",
            "question_name": "Survey",
            "type": "text"
        },
        {
            "index": 0,
            "question_description": "",
            "min": 0,
            "default": "",
            "max": 1024,
            "required": true,
            "choices": "",
            "new_question": true,
            "variable": "why",
            "question_name": "Why?",
            "type": "text"
        }
    ],
    "name": ""
}
```